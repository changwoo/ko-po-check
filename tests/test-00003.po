# -*-text-*-
# test 3 - 자주 범하는 잘못된 국어 사용
#
msgid ""
msgstr ""
"Project-Id-Version: test 3\n"
"POT-Creation-Date: 2002-07-26 03:40+0900\n"
"PO-Revision-Date: 2002-07-26 03:40+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

# E: "안 함"이 맞음
#: test.c:3
msgid "Not translated"
msgstr "번역 않함"

# E: "안 함"이 맞음
#: test.c:3
msgid "Not Translated"
msgstr "번역 않 함"

# E: "안 됩니다"가 맞음
#: test.c:3
msgid "Default resolution must not be zero."
msgstr "해상도 초기값은 0이 되어서는 않됩니다."

# E: "습니다"가 맞음
#: test.c:3
msgid "No such file or directory"
msgstr "그런 파일이나 디렉토리가 없읍니다"

# E: "~음"이 맞음
#: test.c:3
msgid "There is no spoon"
msgstr "숟가락이 없슴"

# E: "~음"이 맞음
#: test.c:3
msgid "There is spoon"
msgstr "숟가락이 있슴"

# E: "~음"이 맞음
#: test.c:3
msgid "She's gone out of my life.  I was wrong, I'm to blame.  I was so untrue"
msgstr ""
"그녀가 내 삶에서 떠나갔슴.  내가 잘못했슴.  모든 게 내 탓이었슴. 내가 너무 거"
"짓되었슴"
