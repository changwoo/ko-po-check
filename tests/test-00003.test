#!/bin/sh

FILTERS="language/spell"

DIR="`dirname $0`"
NAME="`basename $0 .test`"
BUILDDIR=`pwd`
POFILE=$NAME.po
OUTFILE=$NAME.out

cd $srcdir

TFM=`mktemp`
python $BUILDDIR/../scripts/ko-po-check --moduledir=$srcdir/.. --checksdir=$srcdir/../checks --filters="${FILTERS}" $POFILE > $TFM
diff -u $TFM $OUTFILE
RESULT=$?
rm -f $TFM
exit $RESULT
