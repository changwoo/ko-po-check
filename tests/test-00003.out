test-00003.po:16: "%s가": 받침에 따른 조사 구별(예: %s을(를))이 없습니다
